scan/lib/scan/options.rb

```diff
@@ -130,6 +130,7 @@ def self.available_options
                                      is_string: false,
                                      default_value: false),
         FastlaneCore::ConfigItem.new(key: :code_coverage,
+                                     env_name: "SCAN_GENERATE_CODE_COVERAGE",
```

Comment:
If there is no deeper reason for the naming I am missing, I suspect this could/should be `SCAN_CODE_COVERAGE` as the `env_name` usually seems to be toolname + key name.

Reply:
Heh...  I created it by reading the description instead of the key.  I'll make the change.